src/style.css

```
@@ -0,0 +1,119 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  --small-font-waight: 400;
+  --medium-font-waight: 500;
+  --big-font-waight: 700;
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  padding: 16px;
```

```suggestion
  padding: 36px, 16px, 16px;
```
краще задати карточці падінг а не картинці марджин